### DISCUSSION TITLE
LibreLinkUp sign up without scanning the sensor?

### DISCUSSION THREAD

**Discussion by podupuis** [creepymonster/GlucoseDirect#520] [Q&A]
Is it possible to add an option to have my data inside glucosedirect without having to scan my sensor? Just by loggin in my LibreLinkUp account on glucosedirect? I’m in Canada and unfortunately, the sensor isn’t scanning with glucosedirect.

**Comment by Houbsi** [comment]
Hi @podupuis ,

Would it be possible that you could tell in a detailed description how you scan and what the error message is?

And for what reason does the issue link to your Canadian Location?
